classes/02class/exercises/c02-network06/raghunadhpokkalath/README.md

```diff
@@ -0,0 +1,89 @@
+# c02-network06
+
+## Commands Execution Output
+
+- Commands for creating the NAT gateway:
+```
+Allocate  Elastic Ip address
+
+> aws ec2 allocate-address
+{
+    "PublicIp": "13.236.30.80",
+    "AllocationId": "eipalloc-0806e1b4a4b1c5143",
+    "PublicIpv4Pool": "amazon",
+    "NetworkBorderGroup": "ap-southeast-2",
+    "Domain": "vpc"
+}
+
+Create a natgateway and assoicate with Elastic ip and public subnet
+
+> aws ec2 create-nat-gateway --allocation-id eipalloc-0806e1b4a4b1c5143 --subnet-id subnet-0ff0ad4440d0df64e
+{
+    "ClientToken": "1a204877-589e-44da-ba7b-6c208f54ec7b",
+    "NatGateway": {
+        "CreateTime": "2020-07-05T09:38:00+00:00",
+        "NatGatewayAddresses": [
+            {
+                "AllocationId": "eipalloc-0806e1b4a4b1c5143"
+            }
+        ],
+        "NatGatewayId": "nat-01d6374c190b9261d",
+        "State": "pending",
+        "SubnetId": "subnet-0ff0ad4440d0df64e",
+        "VpcId": "vpc-076a05ec0f3be4895"
+    }
+}
+
+
+Create a tag and associate devopsacademy-ngw
+>aws ec2 create-tags --resources nat-01d6374c190b9261d --tags Key=Name,Value=devopsacademy-ngw
```

Comment:
You could use `--tag-specifications 'ResourceType=natgateway,Tags=[{Key=Name,Value=devopsacademy-ngw}]'` when creating the resource instead of two steps.